src/Form/EmbedButtonForm.php

```diff
@@ -73,6 +73,23 @@ public function form(array $form, FormStateInterface $form_state) {
     $upload_directory = \Drupal::config('entity_embed.settings')->get('upload_directory');
     $upload_location = $file_scheme . '://' . $upload_directory . '/';
 
+    $entity_types = $this->entityManager->getEntityTypeLabels(TRUE);
+    $filtered_entity_types = array();
+    // Add all Content entites by default.
+    $filtered_entity_types['Content'] = $entity_types['Content'];
+    // Select only those config entities which have a view builder.
+    $filtered_config_entities = array();
+    foreach ($entity_types['Configuration'] as $entity_type => $label) {
+      if ($this->entityManager->hasHandler($entity_type, 'view_builder')) {
```

Comment:
What do you suggest should be the default behavior then? Do you have any other (more suitable) criteria for dis-allowing config entities?